File: scripts/commands/generic/cmdGroupDisband.js
function executeCommand(galacticManager, player, target, args) {
    var GroupEventIntent = Java.type("intents.GroupEventIntent");
    var GroupEventType = Java.type("intents.GroupEventIntent.GroupEventType");

    if (target != null && (args == null || args.length == 0)) {
        new GroupEventIntent(GroupEventType.GROUP_DISBAND, player, target).broadcast();
    } else {
        var kickingPlayer = galacticManager.getPlayerManager().getPlayerByCreatureFirstName(args);
        if (kickingPlayer != null)
            new GroupEventIntent(GroupEventType.GROUP_DISBAND, player, kickingPlayer.getCreatureObject()).broadcast();
        else
            new GroupEventIntent(GroupEventType.GROUP_DISBAND, player, null).broadcast();
    }
}

File: src/main/java/com/projectswg/holocore/resources/support/global/commands/callbacks/group/CmdGroupDisband.java
package com.projectswg.holocore.resources.support.global.commands.callbacks.group;

import com.projectswg.holocore.intents.gameplay.player.group.GroupEventIntent;
import com.projectswg.holocore.resources.support.global.commands.ICmdCallback;
import com.projectswg.holocore.resources.support.global.player.Player;
import com.projectswg.holocore.resources.support.objects.swg.SWGObject;

public final class CmdGroupDisband implements ICmdCallback {
	
	@Override
	public void execute(Player player, SWGObject target, String args) {
		new GroupEventIntent(GroupEventIntent.GroupEventType.GROUP_DISBAND, player, null).broadcast();
	}
	
}
